package.json

```
@@ -6,7 +6,7 @@
     "prettier-watch": "onchange '**/*.js' '**/*.json' -- prettier --write {{changed}}",
     "dev": "onchange -i './packages/**/src/**/*.js' -- yarn genJSConfig",
     "genJSConfig": "node ./scripts/genJsconfig",
-    "test": "jest --projects ./packages/*",
+    "test": "jest --projects ./packages/n4s/src/rules",	
```

```suggestion
    "test": "jest --projects ./packages/*",
```
You don't need to change this every time you want to run focused tests. If you want to just run the test for the rules, you can 
```yarn test --watch rules```

or ```yarn test --watch n4s``` to run all the tests for n4s